apps/prairielearn/src/models/course.ts

```diff
@@ -154,3 +154,37 @@ export async function deleteCourse({
     throw new Error('Course to delete not found');
   }
 }
+
+export async function insertCourse({
+  institution_id,
+  short_name,
+  title,
+  display_timezone,
+  path,
+  repository,
+  branch,
+  authn_user_id,
+}: Pick<
+  Course,
+  'institution_id' | 'short_name' | 'title' | 'display_timezone' | 'path' | 'repository' | 'branch'
+> & {
+  authn_user_id: string;
+}): Promise<Course> {
+  return await queryRow(
+    sql.insert_course,
+    {
+      institution_id,
+      short_name,
+      title,
+      display_timezone,
+      path,
+      repository,
+      branch,
+      authn_user_id,
+    },
+    CourseSchema,
+  ).catch((err) => {
+    console.log(err);
```

Comment:
I don't think this belongs in the final code, hopefully this is just left over from debugging?

Reply:
Actually I had added this as a placeholder because the sproc had a special check for uniqueness that would cause a custom exception instead of the default one. I wanted to make sure I used the same kind of custom exception in my code, but turns out that the uniqueness exception would never be triggered (since all unique/primary keys in pl_courses involve fields not used in this update), but I forgot to remove the placeholder. I removed it in aa30bcfd6.